modules/cnf-performing-end-to-end-tests-running-the-tests.adoc

```diff
@@ -10,34 +10,44 @@ Run the cluster latency tests to validate node tuning for your Cloud-native Netw
 
 [NOTE]
 ====
-When executing `podman` commands as a non-root or non-privileged user, mounting paths can fail with `permission denied` errors. To make the `podman` command work, append `:Z` to the volumes creation; for example, `-v $(pwd)/:/kubeconfig:Z`. This allows `podman` to do the proper SELinux relabeling.
+When executing `podman` commands as a non-root or non-privileged user, mounting paths can fail with `permission denied` errors. Depending on your local OS and SELinux configuration, you might also experience issues running these commands from your home directory. To make the `podman` commands work, run the commands from a folder that is not your home directory, and append `:Z` to the volumes creation. For example, `-v $(pwd)/:/kubeconfig:Z`. This allows `podman` to do the proper SELinux relabeling.
 ====
 
 .Procedure
 
 . Open a shell prompt in the directory containing the `kubeconfig` file.
 +
-You provide the test image with a `kubeconfig` file in current directory and its related `$KUBECONFIG` environment variable, mounted through a volume. This allows the running container to use the `kubeconfig` file from inside the container.
-
-. Run the latency tests by entering the following command:
+You provide the test image with a `kubeconfig` file in current directory and its related `$KUBECONFIG` environment variable, mounted through a volume. This allows the running container to use the `kubeconfig` file from inside the container. 
++
+[NOTE]
+====
+In the following command, your local `kubeconfig` is mounted to kubeconfig/kubeconfig in the cnf-tests container, which allows access to the cluster.
+====
++
+. To run the latency tests, run the following command, substituting variable values as appropriate:
 +
 [source,terminal,subs="attributes+"]
 ----
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
--e LATENCY_TEST_RUNTIME=<time_in_seconds>\
--e MAXIMUM_LATENCY=<time_in_microseconds> \
+-e LATENCY_TEST_RUNTIME=300\
+-e MAXIMUM_LATENCY=20 \
```

Comment:
There's a note saying that for valid results the test should run for 12 hours so unsure if you should hardcode 5 mins here?

Reply:
Thanks @rohennes for your feedback. Yes, I have changed it to match 12 hours or (43200 seconds). The individual tests have LATENCY_TEST_RUNTIME=600.

Reply:
We should explicitly document that the number means seconds. 12+ hours are needed for the final verification, but during testing we start with shorter periods.

Reply:
Thanks @Marsik for your feedback. I have spelt out "The LATENCY_TEST_RUNTIME is shown in seconds, in this case 600 seconds (10 minutes)."

I have also changed all 4 instances of the note to read "During testing shorter time periods, as shown, can be used to run the tests. However, for final verification and valid results, the test should run for at least 12 hours (43200 seconds)."